_posts/2017-07-28-reproducible-research-badges.md

```diff
@@ -0,0 +1,154 @@
+author: 'Lukas Lohoff, Daniel Nüst'
+---
+
+*This blog post is based on work started in the study project [Badges for computational geoscience containers](https://zivgitlab.uni-muenster.de/geocontainer-badges) at [ifgi](https://www.uni-muenster.de/Geoinformatics/). We thank the [project team](https://github.com/o2r-project/o2r-badger#contributors) for their great work!*
+
+## Introduction
+
+Today badges are widely used in open source software repositories. They have a high recognition value and consequently provide an easy and efficient way to convey basic metadata such as version numbers, build status, download count, or container image size. Now imagine similar badges not for software projects but for modern research: We developed a backend service that provides badges for reproducible research papers to answer questions such as: 
+
+- When was a research paper published?
+- Is it openly accessible?
+- Was the paper published in a peer reviewed journal?
+- What is the research location i.e. the location where the data is from?
+- Can the results be reproduced?
+
+We are not the first nor the only ones to do this: [ScienceOpen](https://www.scienceopen.com/) already has badges for open access publications, content type, views, comments and [Altmetric](https://www.altmetric.com/) score:
+
+![scienceopen badges](/public/images/2017-07-28-badges/scienceOpen.png "Figure 1: ScienceOpen badges")
+<p class="attributionInlineImage">Figure 1: ScienceOpen badges</p>
+
+
+These are helpful when using the ScienceOpen website, but they’re not available for other websites. Additional issues are the four different styles and that some relevant information for reproducible geosciences is not available, e.g. executability status or the research location.
+
+Badges are also used directly for publications, leaving out the search engine “middleman”. Usually the published document, poster or presentation contains a badge along with the information needed to access the data or code. For example, the [Center for Open Science](https://cos.io/) [designed badges](https://osf.io/tvyxz/wiki/home/) for scientific articles. University of Washington’s [eScience Institute](http://escience.washington.edu/) created a peer-review process for open data and open materials badges (https://github.com/uwescience-open-badges/about) based on the COS badges. The service is meant for faculty members and students at the University of Washington, but even external researchers can apply. They also have a great list of [publications on the topic](https://github.com/uwescience-open-badges/about#where-can-i-read-more-about-this).
+
+A study [[1](#kidwell)] by Mallory C. Kidwell et. al. demonstrates that introducing such open data badges had a positive effect in the *Psychological Science* journal: After the journal started awarding badges for open data, more articles stated open data availability. The. They see badges as a simple yet effective way to promote data publishing. 
```

Comment:
put the citation link behind the "et al.", and it is "et al.", not "et. al.", see https://de.wikipedia.org/wiki/Et_al.